src/frontend/app/contact/cunningham.ts

```diff
@@ -0,0 +1,179 @@
+const config = {
```

Comment:
in commit message mayge `app/contact`?

Reply:
https://github.com/numerique-gouv/people/pull/3/commits/84f94adb00f0e4380ead860e25c7d0ae4e5c2a52